pkg/config/validator.go

```diff
@@ -165,12 +165,12 @@ func (v *Validated) Clone() *Validated {
 }
 
 const (
-	sGlobal      = "global"
-	sScopes      = "scopes"
-	sSubjects    = "subjects"
-	sRules       = "rules"
-	sAdapters    = "adapters"
-	sDescriptors = "descriptors"
+	constGlobal      = "global"
```

Comment:
I don't think that's in the spirit of the naming convention. Perhaps @douglas-reid has suggestions.